common/app/templates/headerInlineJS/shouldEnhance.scala.js

```diff
@@ -33,11 +33,11 @@
         return false;
     };
 
-    var isRetinaIpad = function() {
-        return (navigator.platform === 'iPad' && window.devicePixelRatio === 2);
+    var isIpad = function() {
+        return (navigator.platform === 'iPad');
     };
 
-    window.shouldEnhance = !personPrefersCore() && !isOlderIOSDevice() && !(@item.isFront && isRetinaIpad());
+    window.shouldEnhance = !personPrefersCore() && !isOlderIOSDevice() && !(@item.isFront && isIpad());
```

Comment:
I think we could remove the `olderIPadOnFront` check inside `isOlderIOSDevice` if we do it here instead?